helm-org-ql.el

```diff
@@ -178,21 +178,27 @@ Is transformed into this query:
   "Return Helm source named NAME that searches BUFFERS-FILES with `helm-org-ql'."
   ;; Expansion of `helm-build-sync-source' macro.
   (helm-make-source name 'helm-source-sync
+    :filtered-candidate-transformer (lambda (candidates _)
+				      (let ((window-width (window-width (helm-window))))
```

Comment:
It would probably be better to bind the window width variable around the call to `helm-make-source` so it need not be done on every candidate or filtering call.

Reply:
Done.